File: B/Python/Processor.py
class Processor():

	name = ""
	price = 0 

	def __init__(self, name = "", price = 0):
		self.name = name
		self.price = price

	# methods set and get for name attribute
	def setName(self, name):
		self.name = name

	def getName(self):
		return self.name

    # methods set and get for price attribute
	def setPrice(self, price):
		self.price = price

	def getPrice(self):
		return self.price

File: B/Java/Processor.java
public class Processor
{
	private String name; 
	private int price; 
	
	public Processor(String name, int price)
	{
		this.name = name;
		this.price = price;
	}

	public Processor()
	{

	}

	// methods set for name and price attribute.
	public void setName(String name)
	{
		this.name = name;
	}
	public void setPrice(int price)
	{
		this.price = price;
	}

	// methods get for name and price attribute.
	public String getName()
	{
		return this.name;
	}
	public int getPrice()
	{
		return this.price;
	}

}